### PR TITLE
[vulkan] Test & build macOS 10.15 MoltenVK

### PR DESCRIPTION
**Pull request by bobcao3**
Related issue = #

### CODE REVIEW

File: taichi/backends/vulkan/vulkan_loader.cpp [hunk 1]
```diff
@@ -11,6 +11,80 @@ namespace vulkan {
...
+        for (auto &queue : queue_families) {
+          if (queue.queueFlags & VK_QUEUE_COMPUTE_BIT) {
+            found_device_with_compute = true;
+          }
+        }
+
+        VkPhysicalDeviceProperties properties{};
+        vkGetPhysicalDeviceProperties(physical_device, &properties);
+
+        TI_INFO("Found Vulkan Device {} ({})", i, properties.deviceName);
```
Comment (k-ye) [inline taichi/backends/vulkan/vulkan_loader.cpp:76]:
```suggestion
        TI_DEBUG("Found Vulkan Device {} ({})", i, properties.deviceName);
```

**Review by k-ye — APPROVED** [review]
Cool!

File: taichi/backends/vulkan/vulkan_loader.cpp (hunk 1 above)
Comment (bobcao3) [reply taichi/backends/vulkan/vulkan_loader.cpp:76]: This is in conjunction with #4255 so that the user can know which device corresponds to which ID

Comment (bobcao3) [reply taichi/backends/vulkan/vulkan_loader.cpp:76]: Maybe is there a way to display info without including the source file path?

**Review by ailzhang — APPROVED** [review]
LGTM!

**Pull request merged**